src/Util/ClassSourceManipulator.php

```diff
@@ -208,13 +272,29 @@ public function addMethodBody(Builder\Method $methodBuilder, string $methodBody)
         $methodBuilder->addStmts($nodes);
     }
 
+    public function addMethodParams(Builder\Method $methodBuilder, array $params)
```

Comment:
This should be private (and so, it should be at the bottom of this class)

Reply:
Done